bundles/framework/feedbackService/publisher/FeedbackServicePublisherTool.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Checkbox, LabeledInput, Message } from 'oskari-ui';
+
+const Content = styled('div')`
+    display: flex;
+    flex-direction: column;
+`;
+
+const FeedbackFrom = styled('div')`
+    display: flex;
+    flex-direction: column;
+`;
+
+const StyledCheckbox = styled(Checkbox)`
+    + .ant-checkbox-wrapper {
+        margin-left: 0;
+    }
+`;
+
+export const FeedbackServicePublisherTool = ({ state, controller }) => {
```

Comment:
Maybe rename this to FeedbackServiceForm or similar to make it easier to recognize the React-component for the form from the actual publisher tool js-file.

Also add a comment why it's using both `useState()` AND `controller.updateField()` with something like: 
```
controller.updateField() will re-render the form which makes the field that is being edited loose focus after each keypress -> using useState() to workaround this while waiting for the whole publisher to be written in React.
 ```